examples/MCP/simple_calculator_fastmcp/tests/test_simple_calculator_fastmcp.py

```
@@ -0,0 +1,133 @@
+            async with streamablehttp_client(nat_fastmcp_url) as (
+                    read_stream,
+                    write_stream,
+                    _,
+            ):
+                async with ClientSession(read_stream, write_stream) as session:
+                    await session.initialize()
+                    tools = await session.list_tools()
+                    assert 'calculator__subtract' in (t.name for t in tools.tools)
+                    return
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)